specification/voiceservices/data-plane/Microsoft.VoiceServices/stable/2023-10-01/swagger.json

```
@@ -0,0 +1,1015 @@
+          },
+          {
+            "in": "path",
+            "name": "phoneNumber",
+            "pattern": "^\\+[1-9]\\d+$",
+            "minLength": 3,
+            "maxLength": 16,
+            "required": true,
+            "type": "string",
+            "description": "Phone number in E.164 format."
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "An error response object.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/errorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a number.": {
+            "$ref": "./examples/DeleteNumber.json"
+          }
+        },
+        "tags": [
+          "Number"
+        ],
+        "description": "Removes the specified number from the specified account."
+      },
+      "get": {
+        "operationId": "Number_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "accountName",
+            "pattern": "^([A-Za-z0-9_-]+)$",
+            "minLength": 1,
+            "maxLength": 100,
+            "required": true,
+            "type": "string",
+            "description": "Name of the account. Can only contain letters, numbers, underscores and dashes. Can be up to 100 characters in length."
+          },
+          {
+            "in": "path",
+            "name": "phoneNumber",
+            "pattern": "^\\+[1-9]\\d+$",
+            "minLength": 3,
+            "maxLength": 16,
+            "required": true,
+            "type": "string",
+            "description": "Phone number in E.164 format."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/NumberResponseDto"
+            }
+          },
+          "default": {
+            "description": "An error response object.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/errorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a number.": {
+            "$ref": "./examples/GetNumber.json"
+          }
+        },
+        "tags": [
+          "Number"
+        ],
+        "description": "Gets the details of a number assigned to an account, including the services for which the number is enabled."
+      },
+      "put": {
+        "operationId": "Number_Put",
```

The operationId should be descriptive of the operation and not the underlying HTTP method used.
```suggestion
        "operationId": "Number_CreateOrReplace",
```

---

Done. Also checked the other operationsIds (apart from those flagged, the batch endpoint also needed some tweaking).